etc/sbps-marshal.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=System service which reports information about a booted node state
```

Comment:
Replace with 'System service that manages Squashfs images projected via iSCSI for IMS, PE, and other ancillary images similar to PE.'

Reply:
Done. Also updated RPM spec and README.